Core/Utilities/Distribution.py

```diff
@@ -1,464 +1,85 @@
+  versionStrings = ("majorVersion", "minorVersion", "patchLevel", "preVersion")
   reList = []
-  for iP in range( len( versionStrings ) ):
+  for iP, version in enumerate(versionStrings):
     if verTup[iP]:
-      replStr = "%s = %s" % ( versionStrings[iP], verTup[iP] )
+      replStr = "%s = %s" % (versionStrings[iP], verTup[iP])
     else:
       replStr = "%s = 0" % versionStrings[iP]
-    reList.append( ( re.compile( r"^(%s\s*=)\s*[0-9]+\s*" % versionStrings[iP] ), replStr ) )
+    reList.append((re.compile(r"^(%s\s*=)\s*[0-9]+\s*" % versionStrings[iP]), replStr))
   newData = []
-  for line in fileData.split( "\n" ):
+  for line in fileData.split("\n"):
     for reCm, replStr in reList:
-      line = reCm.sub( replStr, line )
-    newData.append( line )
+      line = reCm.sub(replStr, line)
+    newData.append(line)
   try:
-    fd = open( initFile, "w" )
-    fd.write( "\n".join( newData ) )
-    fd.close()
+    with open(initFile, "w") as fd:
+      fd.write("\n".join(newData))
   except Exception as e:
-    return S_ERROR( "Could write to %s: %s" % ( initFile, str( e ) ) )
+    return S_ERROR("Could write to %s: %s" % (initFile, str(e)))
   return S_OK()
 
 
-#
-
-def createTarball( tarballPath, directoryToTar, additionalDirectoriesToTar = None ):
-  tf = tarfile.open( tarballPath, "w:gz" )
-  tf.add( directoryToTar, os.path.basename( os.path.abspath( directoryToTar ) ), recursive = True )
-  if isinstance( additionalDirectoriesToTar, basestring ):
-    additionalDirectoriesToTar = [ additionalDirectoriesToTar ]
+def createTarball(tarballPath, directoryToTar, additionalDirectoriesToTar=None):
+  tf = tarfile.open(tarballPath, "w:gz")
+  tf.add(directoryToTar, os.path.basename(os.path.abspath(directoryToTar)), recursive=True)
+  if isinstance(additionalDirectoriesToTar, basestring):
+    additionalDirectoriesToTar = [additionalDirectoriesToTar]
   if additionalDirectoriesToTar:
     for dirToTar in additionalDirectoriesToTar:
-      if os.path.isdir( dirToTar ):
-        tf.add( dirToTar, os.path.basename( os.path.abspath( dirToTar ) ), recursive = True )
+      if os.path.isdir(dirToTar):
+        tf.add(dirToTar, os.path.basename(os.path.abspath(dirToTar)), recursive=True)
   tf.close()
   md5FilePath = False
-  for suffix in ( ".tar.gz", ".gz" ):
-    sLen = len( suffix )
-    if tarballPath[ len( tarballPath ) - sLen: ] == suffix:
+  for suffix in (".tar.gz", ".gz"):
+    sLen = len(suffix)
+    if tarballPath.endswith(suffix):
       md5FilePath = "%s.md5" % tarballPath[:-sLen]
       break
   if not md5FilePath:
-    return S_ERROR( "Could not generate md5 filename" )
-  md5str = File.getMD5ForFiles( [ tarballPath ] )
-  fd = open( md5FilePath, "w" )
-  fd.write( md5str )
+    return S_ERROR("Could not generate md5 filename")
+  md5str = File.getMD5ForFiles([tarballPath])
+  fd = open(md5FilePath, "w")
+  fd.write(md5str)
   fd.close()
   return S_OK()
-
-#Start of release notes
```

Comment:
Why do you delete the release notes related functions ?

Reply:
Sorry, restored.